2024-11-26 10:07:33,127 DEBUG    root            Loaded Command Group: ['gcloud', 'projects']
2024-11-26 10:07:33,186 DEBUG    root            Loaded Command Group: ['gcloud', 'projects', 'add_iam_policy_binding']
2024-11-26 10:07:33,203 DEBUG    root            Running [gcloud.projects.add-iam-policy-binding] with arguments: [--member: "serviceAccount:trigger-account@inspired-photon-441609-k7.iam.gserviceaccount.com", --role: "main_custom_role", PROJECT_ID: "inspired-photon-441609-k7"]
2024-11-26 10:07:33,364 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): cloudresourcemanager.googleapis.com:443
2024-11-26 10:07:33,519 DEBUG    urllib3.connectionpool https://cloudresourcemanager.googleapis.com:443 "POST /v1/projects/inspired-photon-441609-k7:getIamPolicy?alt=json HTTP/1.1" 200 None
2024-11-26 10:07:33,655 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): cloudresourcemanager.googleapis.com:443
2024-11-26 10:07:33,977 DEBUG    urllib3.connectionpool https://cloudresourcemanager.googleapis.com:443 "POST /v1/projects/inspired-photon-441609-k7:setIamPolicy?alt=json HTTP/1.1" 400 None
2024-11-26 10:07:33,980 INFO     ___FILE_ONLY___ ERROR: Policy modification failed. For a binding with condition, run "gcloud alpha iam policies lint-condition" to identify issues in condition.

2024-11-26 10:07:33,981 DEBUG    root            (gcloud.projects.add-iam-policy-binding) INVALID_ARGUMENT: The role name must be in the form "roles/{role}", "organizations/{organization_id}/roles/{role}", or "projects/{project_id}/roles/{role}".
Traceback (most recent call last):
  File "/snap/google-cloud-cli/289/lib/googlecloudsdk/calliope/cli.py", line 998, in Execute
    resources = calliope_command.Run(cli=self, args=args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/289/lib/googlecloudsdk/calliope/backend.py", line 843, in Run
    resources = command_instance.Run(args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/289/lib/googlecloudsdk/command_lib/util/apis/yaml_command_translator.py", line 1382, in Run
    raise ex
  File "/snap/google-cloud-cli/289/lib/googlecloudsdk/command_lib/util/apis/yaml_command_translator.py", line 1375, in Run
    ref, response = self._CommonRun(args)
                    ^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/289/lib/googlecloudsdk/command_lib/util/apis/yaml_command_translator.py", line 455, in _CommonRun
    response = method.Call(
               ^^^^^^^^^^^^
  File "/snap/google-cloud-cli/289/lib/googlecloudsdk/command_lib/util/apis/registry.py", line 341, in Call
    return request_func(global_params=global_params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/289/lib/googlecloudsdk/command_lib/util/apis/registry.py", line 404, in RequestFunc
    return method(request, global_params=global_params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/289/lib/googlecloudsdk/generated_clients/apis/cloudresourcemanager/v1/cloudresourcemanager_v1_client.py", line 966, in SetIamPolicy
    return self._RunMethod(
           ^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/289/lib/third_party/apitools/base/py/base_api.py", line 747, in _RunMethod
    return self.ProcessHttpResponse(method_config, http_response, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/289/lib/third_party/apitools/base/py/base_api.py", line 753, in ProcessHttpResponse
    self.__ProcessHttpResponse(method_config, http_response, request))
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/289/lib/third_party/apitools/base/py/base_api.py", line 612, in __ProcessHttpResponse
    raise exceptions.HttpError.FromResponse(
apitools.base.py.exceptions.HttpBadRequestError: HttpError accessing <https://cloudresourcemanager.googleapis.com/v1/projects/inspired-photon-441609-k7:setIamPolicy?alt=json>: response: <{'vary': 'Origin, X-Origin, Referer', 'content-type': 'application/json; charset=UTF-8', 'content-encoding': 'gzip', 'date': 'Tue, 26 Nov 2024 10:07:33 GMT', 'server': 'ESF', 'cache-control': 'private', 'x-xss-protection': '0', 'x-frame-options': 'SAMEORIGIN', 'x-content-type-options': 'nosniff', 'server-timing': 'gfet4t7; dur=280', 'transfer-encoding': 'chunked', 'status': 400}>, content <{
  "error": {
    "code": 400,
    "message": "The role name must be in the form \"roles/{role}\", \"organizations/{organization_id}/roles/{role}\", or \"projects/{project_id}/roles/{role}\".",
    "status": "INVALID_ARGUMENT"
  }
}
>
2024-11-26 10:07:33,990 ERROR    root            (gcloud.projects.add-iam-policy-binding) INVALID_ARGUMENT: The role name must be in the form "roles/{role}", "organizations/{organization_id}/roles/{role}", or "projects/{project_id}/roles/{role}".
